docs/site/README.md

```diff
@@ -1,31 +1,55 @@
-## OCIO Homepage Setup / Dev
+# OCIO Homepage Development
 
-# Easy Setup (Hugo + Github Pages)
+The OCIO homepage is built with Hugo which you will need to build the static HTML served from the gh-pages branch.
 
-### What you need !!
+## Building For Development
 
-### Step 1 : Fork or Clone repository - follow OCIO dev guidelines
+**Step 1:** Fork or clone the repository, follow [OCIO contribution guidelines](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/master/CONTRIBUTING.md) for branch names!
```

Comment:
Worth calling out that you should develop on a branch in your fork here? I know it's in the overall contributing guide, but in case people don't read, should be explicit.